yelp_package/Makefile

```diff
@@ -36,11 +36,11 @@ build_%_docker:
 	[ -d ../dist ] || mkdir ../dist
 	docker pull "yelp/paastatools_$*_container" || true
 	cd dockerfiles/$*/ && docker build --build-arg DOCKER_REGISTRY=$(DOCKER_REGISTRY) \
-        --build-arg PIP_INDEX_URL=$(PIP_INDEX_URL) -t "yelp/paastatools_$*_container" .
+		--build-arg PIP_INDEX_URL=$(PIP_INDEX_URL) -t "yelp/paastatools_$*_container" .
 	if [ "$$TRAVIS_BRANCH" = "master" -a \
-	     "$$TRAVIS_PULL_REQUEST" = "false" -a \
-	     "$$DOCKER_USERNAME" != "" ]; then \
-	  docker push "yelp/paastatools_$*_container"; \
+		"$$TRAVIS_PULL_REQUEST" = "false" -a \
+		"$$DOCKER_USERNAME" != "" ]; then \
+		docker push "yelp/paastatools_$*_container"; \
 	fi
```

Comment:
FWIW I think all the Travis build stuff has been removed in favor of GitHub Actions, so this can probably be removed?

Reply:
I didnt even notice the travis part, just that VS Code was showing mixed tab and space indents, but I've removed it now